### ISSUE TITLE
[Feature Request] Expose WithClientCapabilities in TokenAcquisitionOptions to declare the client supports CAE / AuthenticationContext

### ISSUE THREAD

**Issue by jmprieur** [AzureAD/microsoft-identity-web#1071]
**Is your feature request related to a problem? Please describe.**
Clients (web apps or web APIs calling web APIs) need to support CAE and AuthenticationContext

**Describe the solution you'd like**
For this we'd want to expose  `ClientCapabilities` in `TokenAcquisitionOptions`

**See also**
https://docs.microsoft.com/azure/active-directory/develop/app-resilience-continuous-access-evaluation

**Comment by jmprieur** [comment]
This is actually possible because `ApplicationOptions` expose the `ClientCapabilities` property
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/7f9974cc0525d308e091dbf380ef6d8c75aad8d9/src/client/Microsoft.Identity.Client/AppConfig/ApplicationOptions.cs#L123

so the way to use client capabilities would be, in the appsettings.json to set

```Json
"AzureAD" : 
{
 // usual members
 "ClientCapabilities" : [  "cp1" ]
}
```

**Comment by jmprieur** [comment]
cc: @kalyankrishna1 as you were asking about this

**Comment by jennyf19** [comment]
Included in [1.9.0 release](https://github.com/AzureAD/microsoft-identity-web/releases/tag/1.9.0)
